client/src/screens/groups/components/GroupDetailsRenderItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Alert,
+} from 'react-native';
+
+import EventListItem from '../../../components/Events/EventListItem';
+import ScheduleListItem from '../../../components/Schedules/ScheduleListItem';
+import UserListItem from '../../../components/Users/UserListItem';
+
+const GroupDetailsRenderItem = ({ item }) => {
```

Comment:
why don't you just put this function at the top of `GroupDetailsList`? Either in the render function or on the line before it? This isn't a component so it feels weird to have it as a separate file, especially when only used with `GroupDetailsList`.

Reply:
Done